src/main/Mainpage.hpp

```diff
@@ -22,14 +22,12 @@
  *
  * @section intro Introduction
  *
- * Clio is an XRP Ledger API server. Clio is optimized for RPC calls, over WebSocket or JSON-RPC.
+ * Clio is an XRP Ledger API server optimized for RPC calls over WebSocket or JSON-RPC.
  *
- * Validated historical ledger and transaction data are stored in a more space-efficient format, using up to 4 times
- * less space than rippled.
+ * It stores validated historical ledger and transaction data in a more space efficient format, and uses up to 4 times less space than <A HREF="https://github.com/XRPLF/rippled">rippled</A>.
  *
- * Clio can be configured to store data in Apache Cassandra or ScyllaDB, allowing for scalable read throughput.
- * Multiple Clio nodes can share access to the same dataset, allowing for a highly available cluster of Clio nodes,
- * without the need for redundant data storage or computation.
+ * Clio can be configured to store data in  <A HREF="https://cassandra.apache.org/_/index.html">Apache Cassandra</A> or <A HREF="https://www.scylladb.com/">ScyllaDB</A>, enabling scalable read throughput.
+ * Multiple Clio nodes can share access to the same dataset, which allows for a highly available cluster of Clio nodes without the need for redundant data storage or computation.
  *
  * You can read more general information about Clio and its subsystems from the `Related Pages` section.
```

Comment:
Not entirely sure where the `Related Pages` section is here, so I didn't add a link, but would be good to know what we mean here.

Reply:
I think this is outdated now. In doxygen, there was a `Related Pages` link on the top bar that is now disabled entirely. That section basically had links to all the README.md files in each of the components' subfolders.